src/Diagnostics.HealthChecks/Readme.md

```diff
@@ -105,3 +105,23 @@ public class CustomHealthCheck : IHealthCheck
 ```
 
 This approach offers more flexibility and customisation, but it introduces more complexity and coupling with this project.
+
+### Certificates Health Check
+
+The Certificates HealthCheck verifies whether certificates have been loaded correctly. To use it, the `CertificatesHealthCheck` class needs to be registered. Here's an example of how to register it using the `AddTypeActivatedCheck` method:
```

Comment:
Please add more detail regarding what it verifies.
It also validates that the cert's validity period and private key inclusion.

Reply:
I updated the readme and included the description of what exactly it verifies